src/commands/migrate.ts

```diff
@@ -0,0 +1,23 @@
+import { Command, Signup } from '../types'
+import { PermissionFlagsBits } from 'discord.js'
+
+module.exports = new Command({
+	name: 'migrate',
+	description: 'Matches DB object to TS type',
+	allowedChannels: ['bot-commands'],
+	allowedPermissions: PermissionFlagsBits.Administrator,
+	async execute({ ia, mongoSignups }) {
+		const guildMembers = await ia.guild.members.fetch()
+
+		guildMembers.forEach(async member => {
```

Comment:
Might be worth adding a `await ia.deferReply()` here and editing it after? Depends how long it'll take on the main DB (unless you've already done it!)

Reply:
I did it on the dev DB where i copied all the data, and it didn't take too long, and it's also a one time use only. But good point